tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -1250,6 +1250,22 @@ public void TestCustomFunction2()
             Assert.AreEqual(6.0, evaled);
         }
 
+        [TestMethod]
+        public void TestInjectLG()
+        {
+            var templates = Templates.ParseFile(GetExampleFilePath("./InjectionTest/inject.lg"));
+            
+            var evaled = templates.EvaluateText("${foo.bar()}");
+            
+            Assert.AreEqual("3", evaled.ToString());
+
+            evaled = templates.EvaluateText("${foo.cool(2)}");
+            Assert.AreEqual("3", evaled.ToString());
+
+            evaled = templates.EvaluateText("${common.looking()}");
+            Assert.AreEqual("John", evaled);
+        }
+
```

Comment:
Add more Exception tests. 1.parameters is not match and so on

Reply:
make sense, added exception tests